src/message/mod.rs

```diff
@@ -91,27 +92,33 @@ pub trait AsRawIRC {
 /// for the message format that this is based on.
 /// Further, this implements [IRCv3 tags](https://ircv3.net/specs/extensions/message-tags.html).
 #[derive(Debug, Clone, PartialEq, Eq)]
-#[cfg_attr(feature = "with-serde", derive(Serialize, Deserialize))]
+#[cfg_attr(
+    feature = "with-serde",
+    derive(
+        Serialize,
+        Deserialize
+    )
+)]
 pub struct IRCMessage {
     /// A map of additional key-value tags on this message.
     pub tags: IRCTags,
     /// The "prefix" of this message, as defined by RFC 2812. Typically specifies the sending
     /// server and/or user.
     pub prefix: Option<IRCPrefix>,
     /// A command like `PRIVMSG` or `001` (see RFC 2812 for the definition).
-    pub command: String,
+    pub command: FastStr,
```

Comment:
Of course you did, for example here and approx. 80% of the files changed are the message data types.